core/domain/src/main/java/org/the_chance/honeymart/domain/usecase/LoginOwnerUseCase.kt

```diff
@@ -20,6 +20,8 @@ class LoginOwnerUseCase @Inject constructor(
         } else {
             val token = authRepository.loginOwner(email, password)
             authRepository.saveToken(token)
+            authRepository.saveOwnerName("")
```

Comment:
Is the regular case that we save the owner name and imageUrl as empty string? 
Or why is a better question?

Reply:
Nope
We've been told that the login endpoint will respond with the owner info not just the token. 
we just keeping it empty till the login is refactored.